public/index.js

```diff
@@ -78,7 +82,11 @@ buttons.forEach((button) => {
           console.error('Error: Element with class "gpt-response" not found');
         }
       })
-      .catch((error) => console.error("Error:", error));
+      .catch((error) => console.error("Error:", error))
+      .finally(() => {
```

Comment:
Added a .finally which - once the recipe as loaded - triggers the loader to disappear. 
@josueJURE